doc/Language/unicode_entry.pod6

```diff
@@ -103,6 +103,13 @@ Further information about entering special characters in Vim can be found on
 the Vim Wikia page about L<entering special
 characters|http://vim.wikia.com/wiki/Entering_special_characters>.
 
+=head3 vim-perl6
+
+The L<https://github.com/vim-perl/vim-perl6 | vim-perl6> plugin for Vim can be
```

Comment:
Shouldn't this be `L<vim-perl6|https://github.com/vim-perl/vim-perl6>`?

Reply:
You are right.